plugins/coral-plugin-offtopic/client/containers/OffTopicCheckbox.js

```diff
@@ -0,0 +1,11 @@
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+import {addTag, removeTag} from 'coral-plugin-commentbox/actions';
+import OffTopicCheckbox from '../components/OffTopicCheckbox';
+
+const mapStateToProps = ({commentBox}) => ({commentBox});
```

Comment:
We should use `selectors` that we export inside our `coral-plugin-api` to decouple the internal redux state from plugins.